src/containers/shared/SocketContext.tsx

```diff
@@ -23,7 +23,19 @@ function getSocket(rippledUrl?: string): XrplClient {
       `${prefix}://${rippledHost}:443`,
     ])
   }
-  const socket = new XrplClient(wsUrls)
+
+  if (process.env.VITE_RIPPLED_SECONDARY) {
+    process.env.VITE_RIPPLED_SECONDARY.split(',').forEach((network) => {
+      wsUrls.push.apply(wsUrls, [
+        `${prefix}://${network}:${process.env.VITE_RIPPLED_WS_PORT}`,
+        `${prefix}://${network}:443`,
+      ])
+    })
+  }
+
+  const socket = new XrplClient(wsUrls, {
+    tryAllNodes: true,
```

Comment:
Wasn't this causing problems? Were those fixed?

Reply:
They were fixed in 2.0.3